<launch>


  <!--<node name="ardrone_driver" pkg="ardrone_autonomy" type="ardrone_driver" >
    <param name="realtime_navdata" value="true" />
  </node> -->

 

  <node name="ArdroneFollower" pkg="controller" type="ArdroneFollower.py" output="screen">
  <!-- Configures the joystick button mapping -->
		<param name="ButtonEmergency"	value="0" />
		<param name="ButtonLand"		value="1" />
		<param name="ButtonTakeoff"		value="2" />

		<!-- Configures the joystick axis mapping -->
		<param name="AxisRoll"			value="4" />
		<param name="AxisPitch"			value="5" />
		<param name="AxisYaw"			value="2" />
		<param name="AxisZ"				value="3" />

		<!-- Configures the joystick mapping -->
		<param name="ScaleRoll"			value="1" />
		<param name="ScalePitch"		value="1" />
		<param name="ScaleYaw"			value="1" />
		<param name="ScaleZ"			value="1" />
	</node>

  <node name="ArdroneController" pkg="controller" type="ArdroneController.py" output="screen" > 

</node>

 <node name="joy_node" pkg="joy" type="joy_node" clear_params="true">
    <param name="dev" value="/dev/input/js0" />
  </node>

</launch>
